CMake/cdat_modules/cmor_pkg.cmake

```diff
@@ -1,7 +1,9 @@
 set(CMOR_VERSION 2.9.2)
-set(CMOR_URL ${LLNL_URL})
-set(CMOR_GZ CMOR-${CMOR_VERSION}.tar.gz)
-set(CMOR_MD5 f7e78104878d93ffbd3cae2ccd95db2c)
+set(CMOR_BRANCH master)
```

Comment:
Instead of using the `master` branch, it would be better to set a specific commit as the SHA-1 sum. That way, changes in the upstream repository will not break UV-CDAT